.github/pull_request_template.md

```diff
@@ -1,23 +1,33 @@
+# Pull Requests
+
+A Pull Request should close a **single** issue; multiple issues can be closed when the issues are **small and related**, but that should be an exception not the rule. Please
+keep Pull Requests small and targeted; large 'code drops' with dozens of files will be closed and asked to split into reviewable pieces. Reviews that need to be large due to dependencies will be
+reviewed on a case-by-case basis.
+
 ## Related Issue
+
 Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
-the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
+the issue fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one to aid
 in issue tracking.
 
 **NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
 page above.
 
 ## Description
-Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
+
+Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/).
```

Comment:
I've always had a problem with this. The issue being fixed is already described in the issue itself, and it is referenced in the "Related Issue" section above. It doesn't make sense to me to repeat what's already in the issue.

This description should be about how the issue was fixed, and should only be needed when:
- The fix is not straightforward
- The fix introduces new APIs that are not mentioned in the issues
- The fix only partially addresses the issue
- And likely other reasons

Reply:
Seems reasonable to me...changing to ask folks to provide implementation summary of what their fix actually does instead of re-iterating what the problem is that they're fixing